.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
         environment:
           ## this enables colors in the output
           TERM: xterm
+          ipc: host ## https://github.com/cypress-io/cypress/issues/350
```

Comment:
Visit the web page in the comment to understand what this is about. We hit that error multiple times before. Hoping this ends those failures.